### ISSUE TITLE
Simulation Start day

### ISSUE THREAD

**Issue by yyuanjoy** [lbl-srg/EnergyPlusToFMU#13]
Hi Thierry,

The E+ plus fmu seems always started from Jan 1, although I put the begin month as July. How should I do if I only want to simulate the summer condition?

Best,
Yuan

**Comment by tsnouidui** [comment]
Did you solve this issue? You just need to give the begin month and day in seconds.

**Comment by yyuanjoy** [comment]
No, not yet. You mean i put the same seconds number for both begin month and begin day? for example, if i want to start from the second day of Jan, I use the inputs as below?

    86500,                       !- Begin Month
    86400,                       !- Begin Day of Month

**Comment by tsnouidui** [comment]
The master algorithm is in charge of passing the start and stop time of the simulation to the FMU. The times passed by the master will overwrite whatever exists in the IDF.
